rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/service/LibraryService.kt

```diff
@@ -69,5 +77,11 @@ class LibraryServiceImpl(
             Mono.error(LibraryIsMissingException("library id was not found when try to get all words from it"))
         )
         .flatMapMany(libraryRepository::getAllWordsFromLibrary)
+        .doOnNext { GetAllWordsFromLibraryNatsController.log.info("get words {}", it) }
         .map(Word::toDtoResponse)
+
+    override fun getLibraryById(
```

Comment:
I'd like to see get-methods as fail-fast methods. 
In case when no library found here should fall error otherwise name method as findLibraryById